testresultparser/junit_reports.go

```diff
@@ -3,115 +3,134 @@ package testresultparser
 import "encoding/xml"
 
 type report struct {
-    TestSuite   string     `xml:"name,attr"`
-    Tests       int        `xml:"tests,attr"`
-    Failures    int        `xml:"failures,attr"`
-    Errors      int        `xml:"errors,attr"`
-    Skipped     int        `xml:"skipped,attr"`
-    Time        string     `xml:"time,attr"`
-    TestResults []testCase `xml:"testcase"`
+	TestSuite   string     `xml:"name,attr"`
+	Tests       int        `xml:"tests,attr"`
+	Failures    int        `xml:"failures,attr"`
+	Errors      int        `xml:"errors,attr"`
+	Skipped     int        `xml:"skipped,attr"`
+	Time        string     `xml:"time,attr"`
+	TestResults []testCase `xml:"testcase"`
+	StdOut      systemOut  `xml:"system-out"`
```

Comment:
Why do we need extra struct for those two entries? I think ``xml:"system-out",chardata` should work.